packages/phone-sensors/README.md

```diff
@@ -0,0 +1,181 @@
+
+  startDetectingSensorChangesTask(PhoneSensor.MAGNETOMETER, { sensorDelay: SensorDelay.NORMAL, batchSize: 50 }),
+  // startDetectingPhoneMagnetometerChanges
+
+];
+```
+
+> **Warning**: the data collection for a `PhoneSensor` can only be started once, if `startDetectingPhoneFastAccelerometerChanges` is
+> executed after `startDetectingPhoneAccelerometerChanges` and while the collection is in progress, `startDetectingPhoneFastAccelerometerChanges`
+> will be ignored. Therefore, if you want to dynamically change the collection's configuration while the collection is in progress, 
+>  you will have to stop the collection to start it again with the new desired configuration. 
+
+Task generator parameters:
+
+| Parameter name          | Type                    | Description                                        |
+|-------------------------|-------------------------|----------------------------------------------------|
+| `sensor`                | `PhoneSensor`           | Sensor to collect data from. See below.            |
+| `providerConfiguration` | `ProviderConfiguration` | Collection's configuration of the task. See below. |
+
+- PhoneSensor
+
+| Value           | Description                                  |
+|-----------------|----------------------------------------------|
+| `ACCELEROMETER` | Represents the phone's accelerometer sensor. |
+| `GYROSCOPE`     | Represents the phone's gyroscope sensor.     |
+| `MAGNETOMETER`  | Represents the phone's magnetometer sensor.  |
+
+- ProviderConfiguration
+
+| Property      | Type                                   | Description                                                                                                                                           |
+|---------------|----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `sensorDelay` | <code>SensorDelay &vert; number</code> | Indicates the time between two consecutive samples. It can be a `SensorDelay` (i.e., `UI`, `NORMAL`, `GAME` or `FASTEST`) or a value in milliseconds. |
+| `batchSize`   | `number`                               | Indicates the amount of samples to be delivered in each record.                                                                                       |
+
+> **Note**: the `sensorDelay` is taken as a suggestion by the Android OS. Samples could be delivered at a smaller or higher rate.
+
+Task output events:
+
+> These tasks don't produce significant events after they complete their execution aside from the regular `{task-name}Finished` events.
+>
+> However, once the start task has finished running, relevant events will be emitted by the internal listeners while the data collection is active. These are listed below.
+
+- [`accelerometerSamplesAcquired`](#events)
+- [`gyroscopeSamplesAcquired`](#events)
+- [`magnetometerSamplesAcquired`](#events)
```

Comment:
Mmmm... I forgot about it. Well, given that the tasks have the infix `Phone`, i wouldn't see with bad eyes to add the prefix to the events: `phoneAccelerometerSamplesAcquired`, etc... 

I don't know, as almighty developer of this framework, you choose 😉.

Reply:
😂😂😂

I think I would follow the same convention as with the geolocation, wifi and ble packages. State the kind of device that will run the "asignment" in the name of the task, but since the phone is the "main" actor, no need to include in each event where the data comes from. However, if at some point it comes from another device, we should state it to make a distinction.